rubin_sim/scheduler/utils/utils.py

```diff
@@ -182,10 +187,11 @@ def restore_scheduler(
     observatory.mjd = (
         obs["mjd"] + observatory.observatory.visit_time(obs) / 3600.0 / 24.0
     )
+    observatory.obs_id_counter = obs["ID"] + 1
     observatory.observatory.parked = False
-    observatory.observatory.current_RA_rad = obs["RA"]
+    observatory.observatory.current_ra_rad = obs["RA"]
     observatory.observatory.current_dec_rad = obs["dec"]
-    observatory.observatory.current_rotSkyPos_rad = obs["rotSkyPos"]
+    observatory.observatory.current_rot_sky_pos_rad = obs["rotSkyPos"]
     observatory.observatory.cumulative_azimuth_rad = obs["cummTelAz"]
     observatory.observatory.mounted_filters = filters_needed
```

Comment:
I think you might also need to set observatory.observatory.current_filter to the one used in the last observation.